### PR TITLE
git kill with unknown parent

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo @allewun @ricmatsui

### CODE REVIEW

File: features/step_definitions/branch_steps.rb [hunk 1]
```diff
@@ -16,6 +16,11 @@
 end
 
 
+Given(/^I have a( local)?(?: feature)? branch named "([^"]+)" with no parent$/) do |local, branch_name|
+  create_branch branch_name, remote: !local
```
Comment (kevgo) [inline features/step_definitions/branch_steps.rb]: That negation of local is so cool!

File: src/git-kill [hunk 2]
```diff
@@ -13,10 +13,11 @@ function branch_to_kill {
 
 function preconditions {
   target_branch=$(branch_to_kill "$@")
-  parent_branch_name=$(parent_branch "$target_branch")
-
   ensure_is_feature_branch "$target_branch" "You can only kill feature branches."
 
+  ensure_knows_parent_branches "$target_branch"
```
Comment (kevgo) [inline src/git-kill]: The spec sounds like it applies to all commands, yet the implementation is only for git-kill. Are you planning to add scenarios for the other commands later? Or do we want to move the spec into the folder for git-kill?

Comment (charlierudolph) [inline src/git-kill]: The plan is to add feature tests for each command that needs it into here. Planning on moving the git sync, and git ship tests in here as well

Comment (kevgo) [inline src/git-kill]: okay, sounds good

**Comment by kevgo** [comment]
:ship:

**Pull request closed without merging**